data/aurora/VTBB.lartcc

```diff
@@ -0,0 +1,1117 @@
+T;VTBB_NE_TMA;N013.48.08.908;E101.26.51.668;
+T;VTBB_NE_TMA;N013.47.17.488;E101.26.45.356;
+T;VTBB_NE_TMA;N013.46.26.181;E101.26.38.141;
+T;VTBB_NE_TMA;N013.45.35.003;E101.26.30.026;
+T;VTBB_NE_TMA;N013.53.36.800;E100.35.46.300;
+//VTBB_SE_TMA
+T;VTBB_SE_TMA;N013.53.36.800;E100.35.46.300;
+T;VTBB_SE_TMA;N013.45.35.003;E101.26.30.026;
+T;VTBB_SE_TMA;N013.44.44.109;E101.26.21.040;
+T;VTBB_SE_TMA;N013.43.53.372;E101.26.11.163;
+T;VTBB_SE_TMA;N013.43.02.808;E101.26.00.400;
+T;VTBB_SE_TMA;N013.42.12.432;E101.25.48.753;
+T;VTBB_SE_TMA;N013.41.22.258;E101.25.36.227;
+T;VTBB_SE_TMA;N013.40.32.302;E101.25.22.824;
+T;VTBB_SE_TMA;N013.39.42.576;E101.25.08.549;
+T;VTBB_SE_TMA;N013.38.53.098;E101.24.53.407;
+T;VTBB_SE_TMA;N013.38.03.879;E101.24.37.401;
+T;VTBB_SE_TMA;N013.37.14.936;E101.24.20.537;
+T;VTBB_SE_TMA;N013.36.26.283;E101.24.02.820;
+T;VTBB_SE_TMA;N013.35.37.933;E101.23.44.255;
+T;VTBB_SE_TMA;N013.34.49.901;E101.23.24.848;
+T;VTBB_SE_TMA;N013.34.02.201;E101.23.04.604;
+T;VTBB_SE_TMA;N013.33.14.847;E101.22.43.530;
+T;VTBB_SE_TMA;N013.32.27.852;E101.22.21.631;
+T;VTBB_SE_TMA;N013.31.41.231;E101.21.58.915;
+T;VTBB_SE_TMA;N013.30.54.998;E101.21.35.389;
+T;VTBB_SE_TMA;N013.30.09.164;E101.21.11.058;
+T;VTBB_SE_TMA;N013.29.23.745;E101.20.45.931;
+T;VTBB_SE_TMA;N013.28.38.754;E101.20.20.015;
+T;VTBB_SE_TMA;N013.27.54.202;E101.19.53.317;
+T;VTBB_SE_TMA;N013.27.10.104;E101.19.25.846;
+T;VTBB_SE_TMA;N013.26.26.473;E101.18.57.610;
+T;VTBB_SE_TMA;N013.25.43.321;E101.18.28.616;
+T;VTBB_SE_TMA;N013.25.00.660;E101.17.58.874;
+T;VTBB_SE_TMA;N013.24.18.504;E101.17.28.393;
+T;VTBB_SE_TMA;N013.23.36.864;E101.16.57.181;
+T;VTBB_SE_TMA;N013.22.55.753;E101.16.25.247;
+T;VTBB_SE_TMA;N013.22.15.182;E101.15.52.602;
+T;VTBB_SE_TMA;N013.21.35.164;E101.15.19.254;
+T;VTBB_SE_TMA;N013.20.55.710;E101.14.45.214;
+T;VTBB_SE_TMA;N013.20.16.832;E101.14.10.491;
+T;VTBB_SE_TMA;N013.19.38.541;E101.13.35.096;
+T;VTBB_SE_TMA;N013.19.00.848;E101.12.59.039;
+T;VTBB_SE_TMA;N013.18.23.765;E101.12.22.330;
+T;VTBB_SE_TMA;N013.17.47.301;E101.11.44.982;
+T;VTBB_SE_TMA;N013.17.11.468;E101.11.07.003;
+T;VTBB_SE_TMA;N013.16.36.277;E101.10.28.407;
+T;VTBB_SE_TMA;N013.16.01.736;E101.09.49.203;
+T;VTBB_SE_TMA;N013.15.27.858;E101.09.09.404;
+T;VTBB_SE_TMA;N013.14.54.650;E101.08.29.021;
+T;VTBB_SE_TMA;N013.14.22.124;E101.07.48.066;
+T;VTBB_SE_TMA;N013.13.50.288;E101.07.06.551;
+T;VTBB_SE_TMA;N013.13.19.152;E101.06.24.487;
+T;VTBB_SE_TMA;N013.12.48.725;E101.05.41.889;
+T;VTBB_SE_TMA;N013.12.19.015;E101.04.58.767;
+T;VTBB_SE_TMA;N013.11.50.032;E101.04.15.135;
+T;VTBB_SE_TMA;N013.11.21.784;E101.03.31.005;
+T;VTBB_SE_TMA;N013.10.54.278;E101.02.46.390;
+T;VTBB_SE_TMA;N013.10.27.524;E101.02.01.303;
+T;VTBB_SE_TMA;N013.10.01.529;E101.01.15.758;
+T;VTBB_SE_TMA;N013.09.36.300;E101.00.29.767;
+T;VTBB_SE_TMA;N013.09.11.845;E100.59.43.344;
+T;VTBB_SE_TMA;N013.08.48.171;E100.58.56.503;
+T;VTBB_SE_TMA;N013.08.25.285;E100.58.09.258;
+T;VTBB_SE_TMA;N013.53.36.800;E100.35.46.300;

```

Comment:
According to AIP Thailand, there is no northeast and southeast sector. Can you explain this?